python/shotgun_model/shotgun_query_model.py

```diff
@@ -684,20 +681,36 @@ def _load_cached_data(self):
             time_before = time.time()
             num_items = self.__load_from_disk()
             time_diff = (time.time() - time_before)
-            logger.debug(
+            self._log_debug(
                 "Loading finished! Loaded %s items in %4fs" %
                 (num_items, time_diff)
             )
             self.cache_loaded.emit()
             return True
         except Exception, e:
-            logger.debug(
+            self._log_debug(
                 "Couldn't load cache data from disk.\n"
                 " Will proceed with full SG load.\n"
                 "Error reported: %s" % (e,)
             )
             return False
 
+    def _log_debug(self, msg):
```

Comment:
Feels like a recipe that should be part of the framework, something like a class you could derive from that has all the _log_* methods we need and uses .log when available. Let's discuss this in dailies.